understanding/22/consistent-help.html

```diff
@@ -54,6 +54,27 @@ <h2>Intent of Consistent Help</h2>
 
         <p>Without help, some users may abandon the task. They may also fail to correctly complete a task, or they may require assistance from people who do not necessarily keep private information secure.</p>
 
+        <p>It is not the intent of this Success Criterion to require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages. It is also not the intent to require contact information if:</p>
+         <ul>
+            <li>A Web site is not supported,</li>
+				<li>Content is archived, or </li>
+            <li>When finding help would invalidate the activity such as in a testing situation. </li>
+         </ul>
+
+        <p>It is also not the intent of this Success Criterion to require a human be available at all times. If the human contact is not available during certain hours or certain days, information should be provided so the user can tell when it will be available. </p>
+   </section>
+
+   <section id="benefits">
+      <h2>Benefits of Consistent Help</h2>
+
+      <ul>
+         <li>This Success Criterion helps people who may have difficulty locating help that is not part of the page they are using. Ensuring they can connect with help allows them to complete the task.</li>
+         <li>When a user can easily find help for their specific questions, they are more successful completing tasks.</li>
+         <li>Users that experience cognitive fatigue or cognitive shut down will be able to reserve their energy for the task, instead of using it to find support.</li>
+         <li>Enabling users (especially those with cognitive disabilities) to find solutions while expressing their question using their own words increases their chances of success for completing a task.</li>
+      </ul>
+
+      <h2>Additional Benefits of Consistent Help</h2>
         <p>Self help methods beyond the site, such as using internet search to find the contact information for an organization, can be too difficult. Further, the user’s disability may make it more difficult to find the help available (such as a “contact us” link, phone number, or support page) if the information is not present within a few interactions (e.g., displayed in the header, or via a menu). In addition, for some users with disabilities, struggling to complete a task on a site may cause additional cognitive challenges when searching for help within the site.</p>
 
         <p>When a user is quickly able to find help, they are able to complete the task even if they encounter challenges.</p>
```

Comment:
Could I suggest that after this paragraph, an extra heading be added, along the lines of

```
<h2>Help Mechanisms</h2>
```

and, on the following paragraph, changing the first sentence from

```
At least one of the following mechanisms to get help should be included on all websites:
```

to

```
Typical help mechanisms include:
```

(noting that the para after the list includes mention that it would be best for sites to include at least one)

Reply:
This would then also eliminate the contentious "should" word.